server/src/mds3.init.js

```diff
@@ -1814,7 +1814,7 @@ async function validate_query_NIdata(ds, genome) {
 				}
 
 				return new Promise((resolve, reject) => {
-					const ps = spawn('python3', [
+					const ps = spawn(serverconfig.python, [
```

Comment:
should move validate_query_NIdata() from mds3.init.js into server/routes/brainImagingSamples.ts, so to keep the logic together